api/src/main/java/io/minio/ObjectStat.java

```diff
@@ -48,7 +48,8 @@ public ObjectStat(String bucketName, String name, ResponseHeader header, Map<Str
     this.bucketName = bucketName;
     this.name = name;
     this.contentType = header.contentType();
-    this.createdTime = (Date) header.lastModified().clone();
+    Date lastModified = header.lastModified();
+    this.createdTime = lastModified == null ? new Date(-1L) : (Date) header.lastModified().clone();
```

Comment:
Why is `header.lastModified()` `null`? AWS S3 specification does not say the header is optional.

Reply:
@balamurugana Not familiar to AWS S3, we start MINIO server using the docker image with local directory as backend storage (`docker run -p 9000:9000 -e "MINIO_ACCESS_KEY=AKIAIOSFODNN7EXAMPLE" -e "MINIO_SECRET_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY" minio/minio server /XXX/minio`).

Reply:
which version MinIO are you running? @linwukang?

Reply:
> which version MinIO are you running? @linwukang?

almost the latest, minio/minio:RELEASE.2019-07-17T22-54-12Z